vtex-apps/menu
# feat: Change menu item title  name with editorItemTitle

#### What problem is this solving?

Menu Items created on site editor with the same name as "item"

![image](https://user-images.githubusercontent.com/13649073/115081444-a8623680-9eda-11eb-8cf1-5a1173515369.png)

Added the user's possibility to change this with __editorItemTitle field 

![image](https://user-images.githubusercontent.com/13649073/115081590-ed866880-9eda-11eb-9816-32abee699a75.png)

#### How to test it?

Go to the [Workspace](https://itemtitle--storecomponents.myvtex.com/admin/cms/site-editor)

Try to add a new menu item here: 

![image](https://user-images.githubusercontent.com/13649073/115081767-34745e00-9edb-11eb-9b53-c3746425cbc8.png)

![image](https://user-images.githubusercontent.com/13649073/115081804-40f8b680-9edb-11eb-9efe-5c6ffc0b5f9f.png)

Fill Menu Item Name's field, go back, and check if the menu item name created is the same

![image](https://user-images.githubusercontent.com/13649073/115081957-7bfaea00-9edb-11eb-8c5d-50fec380b8ed.png)

#### Related to / Depends on

Zendesk KI: #342030

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/3otPoIR7tGymabNvS8/giphy.gif)
